### DISCUSSION TITLE
Copy Segments

### DISCUSSION THREAD

**Discussion by pacmano1** [nextgenhealthcare/connect#5114] [Q&A]
[IbrahimAlhaj9](https://github.com/IbrahimAlhaj9), I accidentally deleted your thread on copying segments, apologies.  I was trying to delete some of my own comments and goofed.

Can you repost that remaining question?

**Comment by siddharthsharma9258** [comment]
> [IbrahimAlhaj9](https://github.com/IbrahimAlhaj9), I accidentally deleted your thread on copying segments, apologies. I was trying to delete some of my own comments and goofed.
> 
> Can you repost that remaining question?

Original questions can be deleted by anyone? never knew..

**Reply by pacmano1** [reply]
Only because I have collaborator permissions.
